_data/shaunferns26/shaunferns26.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shaun
```

Comment:
Use your full name please, "Shaun Fernandes" or "Shoon-bot" whichever suits you better.